src/components/layout/UserSwitchWrapper.tsx

```diff
@@ -24,24 +28,35 @@ const UserSwitchWrapper: FC<Props> = ({ ButtonContent }) => {
   const { hooks, mutations } = useContext(QueryClientContext);
   const { data: member, isLoading } = hooks.useCurrentMember();
   const { mutateAsync: signOut } = mutations.useSignOut();
+  const { t } = useLibraryTranslation();
 
   return (
-    <Box mr={1}>
-      <GraaspUserSwitch
-        ButtonContent={ButtonContent}
-        signOut={signOut}
-        currentMember={member}
-        isCurrentMemberLoading={isLoading}
-        profilePath={MEMBER_PROFILE_ROUTE}
-        redirectPath={SIGN_IN_ROUTE}
-        renderAvatar={(m) => (
-          <MemberAvatar
-            sx={{ mr: 1 }}
-            size={MEMBER_AVATAR_ICON_SIZE}
-            memberId={m?.id}
-          />
-        )}
-      />
+    <Box sx={{ display: 'flex', gap: '24px', alignItems: 'center' }}>
+      {member?.id && (
+        <Link
+          href={MY_LIST_ROUTE}
+          style={{ textDecoration: 'none', color: 'white' }}
+        >
+          {t(LIBRARY.LIKED_ITEMS)}
+        </Link>
```

Comment:
Can you use the same style as the button "All Collections" ? with the background effect on hover and the font size and weight ? this would make the header more uniform.

Reply:
@spaenleh 
Thanks for your comments, changes are implemented, But I kept the published items tab (check screenshot within PR description), Any recommendations?